CHANGELOG.md

```diff
@@ -6,26 +6,30 @@ This file is used to list changes made in each version of the aws-parallelcluste
 
 2.10.0
 -----
+**ENHANCEMENTS**
+- Add new `all_or_nothing_batch` configuration parameter for `slurm_resume`. If set to `True`, RunInstances calls will
```

Comment:
...for `slurm_resume` *script* ?

When `True`, `slurm_resume` will succeed only if all the instances required by all the pending jobs in Slurm will be available. 
 
^^^ it should be included in the main changelog